semgrep/Pipfile

```diff
@@ -0,0 +1,16 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+pytest = "*"
+wheel = "*"
+tox = "*"
+pytest-snapshot = "*"
+
+[packages]
+semgrep = {editable = true,path = "."}
+
+[requires]
+python_version = "3.7"
```

Comment:
We run tests on 3.6, couldn't we set that here?

Reply:
a bunch of things (nuitka, docker images, etc.) currently rely on 3.7 so keeping it for now